From 0b90ed691fc42971cd039b0f5211bb51f37cf656 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Tue, 24 Jun 2014 15:27:35 +0800
Subject: [PATCH] ENGR00333304 dts: imx6sx: add usdhc into mege_fast wakeup
 source

The usdhc could also be wakeup source, thus add it into the mege_fast
wakeup source list.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
(cherry picked from commit 66223a235ae1d440f382a99ed71e1392d0f09f8b)
---
 arch/arm/boot/dts/imx6sx.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 2ae699a..450bd1a 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -700,6 +700,7 @@
 				compatible = "fsl,imx6sx-gpc", "fsl,imx6q-gpc";
 				reg = <0x020dc000 0x4000>;
 				interrupts = <GIC_SPI 89 IRQ_TYPE_LEVEL_HIGH>;
+				fsl,mf-mix-wakeup-irq = <0x7c00000 0x3d00 0x0 0x400200>;
 			};
 
 			iomuxc: iomuxc@020e0000 {
-- 
1.8.0

